frontend/src/pages/schedule/index.tsx

```diff
@@ -0,0 +1,18 @@
+import Head from "next/head";
+import { useRouter } from "next/router";
+
+export const SchedulePage = () => {
+  const { replace } = useRouter();
+
+  // TODO Replace with _middleware
+  if (typeof window !== "undefined") {
+    replace("/schedule/2022-06-02");
+  }
```

Comment:
as the comment said, I will create a ticket, but there is a bug (?) with nextjs that broke `redirect: {}` so now we can't use it anymore

I want to use _middleware but Apollo raises an error due to undefined __DEV__, which seems to be removed due to tree-shaking. but even updating apollo didn't fix it :(